docs/guides/runtime-options-overview.mdx

```diff
@@ -61,56 +61,56 @@ Scroll to see the full table.
 
 <Tabs>
   <TabItem value="Estimator" label="Estimator">
-| Option               | Sub-option               | Sub-sub-option          | Choices                                                                                   | Default                   | Options              |
+| Option               | Sub-option               | Sub-sub-option          | Choices                                                                                   | Default                   | Option               |
 |----------------------|--------------------------|-------------------------|-------------------------------------------------------------------------------------------|---------------------------|----------------------|
 | [default_shots](../api/qiskit-ibm-runtime/qiskit_ibm_runtime.options.EstimatorOptions)        |                          |         | Integer in the range[0, `backend.max_shots`]| None           | default_shots        |
 | [default_precision](../api/qiskit-ibm-runtime/qiskit_ibm_runtime.options.EstimatorOptions)    |                          |                     |  float > 0     | `0.015625 (1 / sqrt(4096))`     | default_precision    |
 | [dynamical_decoupling](../api/qiskit-ibm-runtime/qiskit_ibm_runtime.options.DynamicalDecouplingOptions) |                          |                         |              |                     | dynamical_decoupling |
-|                      | enable                   |                         | `True`/`False`                                                             |    `False`           |                      |
-|                      | sequence_type            |                         | `XX`/`XpXm`/`XY4`                                                         | `XX`                      |                      |
-|                      | extra_slack_distribution |                         | `middle`/`edges`                                                                  | `middle`                  |                      |
-|                      | scheduling_method        |                         | `asap`/`alap`                                                                      | `alap`                    |                      |
-|                      | skip_reset_qubits        |                         | `True`/`False`                                                                                   | `False`                 |                    |
+|                      | [enable](/api/qiskit-ibm-runtime/qiskit_ibm_runtime.options.DynamicalDecouplingOptions#enable)                   |                         | `True`/`False`                                                             |    `False`           |                      |
```

Comment:
Sorry if I wasn't clear... I don't think we need to link all sub options. Only the ones that are sub categories of options - i.e. the ones that have sub-sub-option. For example, this `enable` is an option by itself, but `layer_noise_learning` is a category that contains more sub-sub-options.